### PR TITLE
Clean config

### PR DESCRIPTION
**Pull request by NadiaHG**
This pull request has to be merged after the https://github.com/ipa320/cob_robots/pull/471

### CODE REVIEW

File: cob_default_robot_config/cob4-1/head_joint_configurations.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-../cob4-2/head_joint_configurations.yaml
```
Comment (floweisshardt) [inline cob_default_robot_config/cob4-1/head_joint_configurations.yaml:1]: as the HW for head is the same for cob4-1 and cob4-2, we sould use softlinks (cob4-2 is original and cob4-1 is linked to cob4-2)

File: cob_default_robot_config/cob4-2/head_joint_configurations.yaml [hunk 2]
```diff
@@ -1,20 +0,0 @@
-joint_names: ["head_1_joint","head_2_joint","head_3_joint"]
```
Comment (floweisshardt) [inline cob_default_robot_config/cob4-2/head_joint_configurations.yaml:1]: use config here and softlink from cob4-1 to cob4-2 (see comment above)

File: cob_default_robot_config/cob4-2/upload_param_cob4-2.launch [hunk 3]
```diff
@@ -4,7 +4,6 @@
 	<!-- send ROBOT parameters to parameter server -->
 	<rosparam command="load" ns="/script_server/base" file="$(find cob_default_robot_config)/cob4-2/base_configurations.yaml"/>
 	<rosparam command="load" ns="/script_server/torso" file="$(find cob_default_robot_config)/cob4-2/torso_joint_configurations.yaml"/>
-	<rosparam command="load" ns="/script_server/head" file="$(find cob_default_robot_config)/cob4-2/head_joint_configurations.yaml"/>
```
Comment (floweisshardt) [inline cob_default_robot_config/cob4-2/upload_param_cob4-2.launch:7]: see comment above

File: cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml [hunk 4]
```diff
@@ -85,10 +85,10 @@ analyzers:
         type: diagnostic_aggregator/GenericAnalyzer
         path: Torso
         contains: 'torso/driver'
-#      head:
-#        type: diagnostic_aggregator/GenericAnalyzer
-#        path: Head
-#        contains: 'head/driver'
+      head:
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-1/config/diagnostics_analyzers.yaml:88]: this should be done for cob4-2, too.

File: cob_hardware_config/cob4-1/config/head_driver.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+sync:
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-1/config/head_driver.yaml:1]: please symlink to cob4-2

File: cob_default_robot_config/cob4-1/head_joint_configurations.yaml (hunk 1 above)
Comment (NadiaHG) [inline cob_default_robot_config/cob4-1/head_joint_configurations.yaml:1]: I wanted to do that but then I saw that the cob4-2 urdf includes a static head: https://github.com/ipa-nhg/cob_robots/blob/indigo_dev/cob_hardware_config/cob4-2/urdf/cob4-2.urdf.xacro#L26 and the head_controllers were only for cob4-1 defined and not as softlinks https://github.com/ipa-nhg/cob_robots/tree/indigo_dev/cob_hardware_config/cob4-1/config , do you know why?

**Comment by NadiaHG** [comment]
This pull request can be closed, if the PR https://github.com/ipa320/cob_robots/pull/473 is right

File: cob_default_robot_config/cob4-1/head_joint_configurations.yaml (hunk 1 above)
Comment (floweisshardt) [inline cob_default_robot_config/cob4-1/head_joint_configurations.yaml:1]: that's an earlier artefact. If the HW is changed now, we should update both (cob4-1 and cob4-2) with a 3dof head by using symlinks.

**Pull request closed without merging**